drivers/unix/dir_access_unix.cpp

```diff
@@ -397,12 +397,18 @@ Error DirAccessUnix::rename(String p_path, String p_new_path) {
 	}
 
 	p_path = fix_path(p_path);
+	if (p_path.ends_with("/")) {
+		p_path = p_path.left(p_path.length() - 1);
```

Comment:
This should be equivalent:
```suggestion
		p_path = p_path.left(-1);
```